### DISCUSSION TITLE
Wildcard variability issues

### DISCUSSION THREAD

**Discussion by temporaryspastic** [adieyal/sd-dynamic-prompts#729] [Q&A]
Hi all!  So I have been having issues with generating a variety of different images using a single wildcard and I am not sure what the issue is, but I believe it may have something to do with the pathways and directories.

For example:

I will write the following prompt: "__bird__" and set the batch count to 2 and I will get the following:
![image](https://github.com/adieyal/sd-dynamic-prompts/assets/139179959/14f3d816-602d-4a10-b093-11b116a979b1)

As you can see, I get 2 images of the same animal - in this case a "Broad-winged hawk".  My goal is to have 2 different birds generated.  Looking at my console during the generation gives me this message:
![image](https://github.com/adieyal/sd-dynamic-prompts/assets/139179959/2a2d8e37-cd28-4e89-a321-2737504ed164)

The issue, though, is that I actually do have that file in that directory.  
![image](https://github.com/adieyal/sd-dynamic-prompts/assets/139179959/4e87c3a9-2aaf-4898-922c-c113771ef500)

I did notice, however, that at some point long ago (I am not sure if I created it myself or it was automatically created) a folder came to being in the \stablediffusionwebui directory called "wildcards".  
![image](https://github.com/adieyal/sd-dynamic-prompts/assets/139179959/6400d933-9e1c-4dec-b433-f2ef8a6a88ba)

In it I had the same .txt files as the C:\stable-diffusion-automatic 1111\stable-diffusion-webui\extensions\sd-dynamic-prompts\wildcards directory.  I have tried deleting that to see if that was the issue, but when that folder was deleted, it would not even make an image with that wildcard despite it still being present the extension directory.  I am at a complete loss at this point and not sure what to do.  I have uninstalled and reinstalled the extension many times, but I think it has something to do with the configuration of the pathways but I just don't know what to do.  Any advice is appreciated!

**Comment by Codyport** [comment]
The confusion of directories indicates an installation problem of some kind.  I would try simplifying - backup or rename the whole A1111 install, install from scratch with the absolute basics necessary, add just the Dynamic Prompt extension and just the wildcard you want.  See if the behaviour is replicated;  if not, build up your install piece by piece and see if it breaks at a given point.  It's a generic approach, not a fix, but in lieu of a specific fix for you, it's the best I can think of.

**Comment by uorufu** [comment]
Do I spot a space in your path? "stable-diffusion-automatic 1111"?

That might be the issue.
